src/env.d.ts

```diff
@@ -8,6 +8,8 @@ interface ImportMetaEnv {
   readonly PUBLIC_PROTECTED_IDS: string;
   readonly PUBLIC_INITIAL_DELIMITER: string;
   readonly PUBLIC_PROTECTED_ID_KEY: string;
+  readonly PUBLIC_ALLOW_SET_THEME: string;
+  readonly PUBLIC_DEFAULT_THEME: string;
```

Comment:
I don't think we need this env var here. The default should be decided by the backend.

Reply:
How do you want the frontend in QA to be able to change themes? Would that also use a backend-only mechanism, or do we need some mechanism on the frontend to do that?